### PR TITLE
Replace pydocstyle with flake8-docstrings in the pre-commit check

### LINKED ISSUE #23: Replace `pydocstyle` with `flake8-docstrings` in the `pre-commit` check
Replace `pydocstyle` with `flake8-docstrings` in the `pre-commit` check

### PR DESCRIPTION
**Pull request by OlenaYefymenko**
Resolve #23

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -64,6 +56,12 @@ repos:
   - id: flake8
     additional_dependencies:
     - flake8-cognitive-complexity ~= 0.1.0
+    - flake8-docstrings
+    args:
+    - --docstring-convention=pep257
```
Comment (webknjaz) [inline .pre-commit-config.yaml:61]: Move this setting to `.flake8`.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -64,6 +56,12 @@ repos:
   - id: flake8
     additional_dependencies:
     - flake8-cognitive-complexity ~= 0.1.0
+    - flake8-docstrings
```
Comment (webknjaz) [inline .pre-commit-config.yaml:59]: Use a more predictable version here.

Comment (OlenaYefymenko) [reply .pre-commit-config.yaml]: Done

File: .pre-commit-config.yaml (hunk 1 above)
Comment (OlenaYefymenko) [reply .pre-commit-config.yaml]: I noticed in the docs that the default is pep257. That's why I deleted it and didn't change `.flake8`.

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -63,7 +55,8 @@ repos:
   hooks:
   - id: flake8
     additional_dependencies:
-    - flake8-cognitive-complexity ~= 0.1.0
+    - flake8-cognitive-complexity == 0.1.0
```
Comment (webknjaz) [inline .pre-commit-config.yaml:58]: Why did you change the version specifier in the unrelated entry? Revert it.

Comment (OlenaYefymenko) [reply .pre-commit-config.yaml:58]: Okay

**Review by webknjaz — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (OlenaYefymenko) [reply .pre-commit-config.yaml]: Marking this as resolved.

File: .pre-commit-config.yaml (hunk 2 above)
Comment (OlenaYefymenko) [reply .pre-commit-config.yaml]: Marking this as resolved.

File: .pre-commit-config.yaml (hunk 3 above)
Comment (OlenaYefymenko) [reply .pre-commit-config.yaml]: Marking this as resolved.
